crates/gpui/Cargo.toml

```diff
@@ -116,6 +116,7 @@ env_logger.workspace = true
 rand.workspace = true
 util = { workspace = true, features = ["test-support"] }
 http_client = { workspace = true, features = ["test-support"] }
+reqwest_client = { workspace = true }
```

Comment:
I have to add this to gpui example, to let remote image works.

Reply:
This does not look good at all, as it's a general library that got some odd, fat client library inside.
Which also uses `tokio` underneath, that is not at all used in Zed which uses gpui.
I understand this is a dev dependency, but Zed uses gpui in tests too, so nope.

We can add a regular `cargo init` thing into `examples` dir and some readme mention that it's a special directory that you run manually, and specify whatever odd dependencies there.
Or hide it behind a feature flag at the very least.

Reply:
I am going to remove this, this PR not requirement of this.